app/interactions/actions/email_check.rb

```diff
@@ -22,82 +24,69 @@ def check_email(parsed_email)
     end
 
     def calculate_check_level
-      Rails.env.test? && check_level == 'smtp' ? :mx : check_level.to_sym
+      check_level_sym = check_level.to_sym
```

Comment:
doesn't need here, you can just return check_level.to_sym without extra variable